l3kernel/l3int.dtx

```diff
@@ -1495,6 +1522,45 @@
 % \end{macro}
 % \end{macro}
 %
+% \begin{macro}
+%   {
+%     \int_set_regex_count:Nnn, \int_set_regex_count:cnn,
+%     \int_gset_regex_count:Nnn, \int_gset_regex_count:cnn
+%   }
+% \begin{macro}
+%   {
+%     \int_set_regex_count:NNn, \int_set_regex_count:cNn,
+%     \int_gset_regex_count:NNn, \int_set_gregex_count:cNn
+%   }
+%    \begin{macrocode}
+\cs_new_protected:Npn \int_set_regex_count:Nnn #1#2#3
+  { \regex_count:nnN {#3} {#2} #1 }
+\cs_generate_variant:Nn \int_set_regex_count:Nnn { c }
+\cs_new_protected:Npn \int_gset_regex_count:Nnn #1#2#3
+  {
+    \group_begin:
+      \int_set_eq:NN \exp_not:N \l_@@_internal_a_int #1
+      \regex_count:nnN {#2} {#3} \l_@@_internal_a_int
+      \int_gset_eq:NN #1 \l_@@_internal_a_int
+    \group_end:
+  }
+\cs_generate_variant:Nn \int_gset_regex_count:Nnn { c }
+\cs_new_protected:Npn \int_set_regex_count:NNn #1#2#3
+  { \regex_count:NnN {#3} {#2} #1 }
+\cs_generate_variant:Nn \int_set_regex_count:NNn { c }
+\cs_new_protected:Npn \int_gset_regex_count:NNn #1#2#3
+  {
+    \group_begin:
+      \int_set_eq:NN \exp_not:N \l_@@_internal_a_int #1
```

Comment:
```suggestion
      \int_set_eq:NN \l_@@_internal_a_int #1
```

Reply:
because Joseph loves \exp_not:N :-)

Reply:
Sigh - I was copying from the 'generated' versions and missed this